spec/requests/users_request_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
```

Comment:
Please make sure spec file should include tests that ensures correct view template is rendered
`For example: expect(response).to render_template(:show)`

Please make sure spec file should include tests that ensures the response body includes the correct placeholder text.
`Example : expect(response.body).to include("Welcome to user index page")`